barretenberg/cpp/src/barretenberg/vm2/constraining/relations/ecc.test.cpp

```diff
@@ -460,5 +476,211 @@ TEST(EccAddConstrainingTest, EccNegativeBadDouble)
     EXPECT_THROW_WITH_MESSAGE(check_relation<ecc>(trace, ecc::SR_OUTPUT_X_COORD), "OUTPUT_X_COORD");
 }
 
+TEST(ScalarMulConstrainingTest, ScalarMulEmptyRow)
+{
+    auto trace = TestTraceContainer::from_rows({
+        { .precomputed_clk = 1 },
```

Comment:
Might be better to have 2 rows,
1. precomputed_first_row = 1
2. precomputed_clk = 1

This helps catch problems, when the relation might use precomputed_first_row. I proposed to @jeanmon that we should have a function in the fixtures.hpp to return this trace and use it in all such tests, but he didn't get to it. Maybe you can add it? then we change the other tests.